### PR TITLE
Porting scenarios to ROS 2.0

### PR DESCRIPTION
**Pull request by ahcorde**
Noteworthy points:

 - The world files try to load a plugin that is not available on these packages, `libgazebo_video_stream_widget.so`, so that prints an error.
 - Bumped all package versions to `2.0.0` to reserve `1.X` for ROS 1
 - Removed the `env_hooks`  and used  `<export>s` in `package.xml` instead.

Worlds:
 - baylands
 - yosemite
 - mcmillan
 - ksql
 - sand_island

### CODE REVIEW

**Comment by chapulina** [comment]
@ahcorde , did you have a chance to look at #7?

**Comment by ahcorde** [comment]
:scream: Sorry I didn't check if something related with ROS 2 was done.

Should I close this PR?

I included the LAT, LON and ALT for each world and the sitl launcher

**Comment by chapulina** [comment]
haha no worries, I was mainly wondering if there were problems with the other PR. I haven't had a chance to compare the two PRs, but feel free to keep this one if it has more stuff than the other one :wink:

**Comment by ahcorde** [comment]
@chapulina  I added some changes based on your commit. I added you as a co-authored ;). Do you think we can close the other PR?

File: mcmillan_airfield/env-hooks/99_mcmillan_airfield_setup.sh.in [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ament_prepend_unique_value GAZEBO_MODEL_PATH "@CMAKE_INSTALL_PREFIX@/share/@PROJECT_NAME@/models"
```
Comment (tfoote) [inline mcmillan_airfield/env-hooks/99_mcmillan_airfield_setup.sh.in]: This is the only instance of a setup file being added back. Is it unneeded here or needed everywhere else?

**Review by tfoote — COMMENTED** [review]
one comment on the setup files. Otherwise it lgtm

File: mcmillan_airfield/env-hooks/99_mcmillan_airfield_setup.sh.in (hunk 1 above)
Comment (ahcorde) [reply mcmillan_airfield/env-hooks/99_mcmillan_airfield_setup.sh.in]: I forgot to remove it

**Comment by ahcorde** [comment]
I forgot to remove it. Done @tfoote

**Review by tfoote — APPROVED** [review]

**Pull request merged**